src/main/java/org/folio/circulation/support/ItemRepository.java

```diff
@@ -16,7 +16,11 @@
 import java.util.function.BiFunction;
 import java.util.stream.Collectors;
 
+import io.vertx.core.json.JsonObject;
```

Comment:
Similar to the comment above, please review the import ordering

Reply:
@marc-johnson , I have the configured formatter, and I applied it again, it added only empty line after:
`import static java.util.function.Function.identity; `

![image](https://user-images.githubusercontent.com/53909129/71824239-bc957200-30a1-11ea-951d-ac3b83a7ce2c.png)

Is that the case?

Reply:
I do not use that plugin so I cannot advise on how to configure it.

My understanding is that the import ordering is java, javax, org, com and then everything else with a line break in between each section.

This change appears to have split the `org` namespace into two separate sections.

Reply:
fixed

Reply:
Please can the imports be fixed for all of the files in this pull request

Reply:
fixed for all files

Reply:
Thank you